### PR TITLE
bs navbar centered, and comment form fixed

### PR DESCRIPTION
**Pull request by lucio-diaz**
Nav bar centered and aligned with the rest of the pages content. The JS call was fixed, after adding a comment, instead of assigning an empty string ("") to each field, a label is assigned to whole form and reseted. Results for validations (required fields) are different.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,15 +15,30 @@
...
+	  <nav class="navbar navbar-expand-lg navbar-dark bg-secondary">
+	    <div class="container-fluid">
+	      <span class="navbar-brand">Blog App</span>
+	    </div>
+	  </nav>
+	</div>
+
+	<div class="col-2">
+	</div>
+      </div>
```
Comment (Ligator) [inline app/views/layouts/application.html.erb:40]: Remove useless code.

Comment (lucio-diaz) [reply app/views/layouts/application.html.erb:40]: I'm triying to center the navbar letting 2 columns to the left, 8 in the center and 2 more to the right, shoud I do it other way?

Comment (lucio-diaz) [reply app/views/layouts/application.html.erb:40]: Triying to align the navbar with the rest of the page. All the pages have the dame code for them do not use the whole width. Si it ok? Should I do IT other way?

Comment (lucio-diaz) [reply app/views/layouts/application.html.erb:40]: Triying to align the navbar with the rest of the page. All the pages have the same code for them do not use the whole width. Si it ok? Should I look for  other way?

Comment (lucio-diaz) [reply app/views/layouts/application.html.erb:40]: About indentation I'll fix it today :)

Comment (Ligator) [reply app/views/layouts/application.html.erb:40]: Remove commented code.

Comment (Ligator) [reply app/views/layouts/application.html.erb:40]: Try to center the navbar with 
https://getbootstrap.com/docs/5.1/layout/grid/#variable-width-content
If it doesn't work, come back to this way but fix the indentation so it won't be confusing.

**Comment by lucio-diaz** [comment]
I tried: row justify-content-md-center, but it gives different result.

**Pull request merged**